GUI/QtQuick/QtQuickAdapter.cpp

```diff
@@ -1848,6 +1857,11 @@ bs::message::ProcessingResult QtQuickAdapter::processWalletDeleted(const std::st
    pushRequest(user_, userWallets_, msg.SerializeAsString());
 
    emit successDeleteWallet();
+
+   if (walletBalances_->rowCount() > 0) {
+	  emit requestWalletSelection(0);
```

Comment:
indentation - it appears to be a tab + 2 spaces this time, should only be 3 spaces per level
same issue below in this file